Equipment.md

```diff
@@ -124,26 +126,30 @@ __Vulnerability__ is an optional rule for players who want excessive realism. It
 
 |Armor|AC Bonus|Move Cost|Vulnerability|
 |---|---|---|---|
-|  Light Leather   |+2 |0 ft. |
+|  Light Leather   |+2 |0 ft. |5 [Slashing]
```

Comment:
While realistic, I hesitate to include these on lower-tier armors. Since early characters will have low-tier and older characters will have high-tier, placing vulnerabilities on high-tier armors is an easy way to:
* Naturally wait to introduce a new mechanic until people are used to the base rules, and
* Help counterbalance the prohibitively good armors.

Reply:
That said, I think adding more medium-tier vulnerabilities could be either good or bad and do not have a strong opinion on that at the moment. Will depend on play. I say leave it for now.